src/fibonacci.js

```diff
@@ -0,0 +1,17 @@
+export default function fibonacci(n){
+  if(n === 1){
+    return [0];
+  }else if(n > 1){
+    var array = [0, 1];
+    var tempArray = [];
+    while(n>2){
+      tempArray = (array.slice(-2));
+      var nextTerm = tempArray.reduce(function(a,b) {return a+b;});
+      array.push(nextTerm);
```

Comment:
Nice use of `splice` and `reduce` to add the last to values together. You could condense it by removing tempArray:
```js
var array = [0, 1];

while (n > 2) { 
  var nextTerm = array.slice(-2).reduce(function(a,b) {return a+b;});
  array.push(nextTerm);
  n--;
}
```